tests/integration.rs

```diff
@@ -2038,6 +2038,41 @@ mod tests {
                 });
         }
 
+        #[test]
+        #[cfg(all(target_arch = "aarch64", feature = "acpi"))]
+        fn test_edk2_acpi_launch() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+
+            vec![Box::new(focal)].drain(..).for_each(|disk_config| {
+                let guest = Guest::new(disk_config);
+
+                let mut child = GuestCommand::new(&guest)
+                    .args(&["--cpus", "boot=1"])
+                    .args(&["--memory", "size=512M"])
+                    .args(&["--kernel", guest.fw_path.as_str()])
```

Comment:
I'd suggest you provide `CLOUDHV_EFI.fd` here, so that you don't have to modify the behavior for all existing tests.

Reply:
I agree this. Now that this is a edk2 specific test case, it's reasonable to use the edk2 specific binary `CLOUDHV_EFI.fd` in this case only.